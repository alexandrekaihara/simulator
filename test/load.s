LOAD TEN
SUB TEN
JMPZ ISZERO
OUTPUT MINUSONE
ISZERO: OUTPUT ONE
STOP
ONE: CONST 1
MINUSONE: CONST -1
TEN: CONST 10
