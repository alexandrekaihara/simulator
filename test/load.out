10 13 2 13 8 8 13 12 13 11 14 1 -1 10
